sellotape/sellotape_dj/chat/tests.py

```diff
@@ -0,0 +1,95 @@
+# chat/tests.py
+from channels.testing import ChannelsLiveServerTestCase
+from selenium import webdriver
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.support.wait import WebDriverWait
+
+
+class ChatTests(ChannelsLiveServerTestCase):
+    serve_static = True  # emulate StaticLiveServerTestCase
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        try:
+            # NOTE: Requires "chromedriver" binary to be installed in $PATH
+            cls.driver = webdriver.Chrome("/Users/tarelgastonmadar/Downloads/chromedriver")
```

Comment:
Hi Tarel,
It seems you've committed a Selenium function to a directory at your own PC, if needed for tests, it should be a part of our project, and referenced respectively.
Does it needed?

Reply:
fixed this issue